brayns/io/CircuitLoader.cpp

```
@@ -307,10 +307,10 @@ class CircuitLoader::Impl
         }
         catch (...)
         {
-            BRAYNS_WARN << "Only MVD2 format is currently supported by Brion "
```

Just print the message if ' Color scheme by layer' was requested? Then it should be an error.

---

Fair enough.

---

Now the commit message and the PR message don't match anymore.

---

Why do you do this check in the try/catch and not before? This hides other exceptions then.